File: LeetCode_3689/Main.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var maxTotalValue = function(nums, k) {
    let mx = Math.max(...nums);
    let mn = Math.min(...nums);
    return (mx - mn) * k;
};

File: LeetCode_3689/Main.c

long long maxTotalValue(int* nums, int n, int k) {
    long long mx = 0, mn = 1000000000;
    for (int i = 0; i < n; i++) {
        if (nums[i] > mx) mx = nums[i];
        if (nums[i] < mn) mn = nums[i];
    }
    return (mx - mn) * k;
}